File: bubblesort.py
def bubbleSort(arr):
    swapped = True
    for i in range(len(arr)):
        swapped = True
        for j in range(0, len(arr) - i - 1):
            if arr[j] > arr[j+1]:
                arr[j], arr[j+1] = arr[j+1], arr[j]
                swapped = False
        if swapped is True:
            break
    return arr


sorted = bubbleSort([2, 1, 5, 99, 7, 10])
print(sorted)

File: bubblesort.js
function bubbleSort(arr) {
    let swapped = true;
    for (let i = arr.length; i > 0; i--) {
        swapped = true;
        for (let j = 0; j < i - 1; j++) {
            if (arr[j] > arr[j + 1]) {
                let temp = arr[j];
                arr[j] = arr[j + 1];
                arr[j + 1] = temp;
                swapped = false
            }
        }
        if (swapped == true) {
            break
        }
    }
    return arr
}

sorted = bubbleSort([2, 1, 5, 99, 6, 10])
console.log(sorted)